src/parser/bison_parser.y

```diff
@@ -526,6 +528,13 @@ create_statement:
 			$$->tableName = $4.name;
 			$$->select = $6;
 		}
+	|	CREATE INDEX opt_index_name opt_not_exists ON table_name '(' ident_commalist ')' {
```

Comment:
`IF NOT EXISTS` is before the index name, i.e., the order should be `opt_not_exists opt_index_name`.
https://www.postgresql.org/docs/current/sql-createindex.html